design-system/src/theme-provider.tsx

```diff
@@ -35,7 +38,7 @@ const toVars = (obj: Record<string, string>) =>
 type ThemeProviderProps = {
   children: ReactNode;
   theme?: string;
-  scope?: 'global' | 'local';
+  customPropertiesOverrides?: Record<string, string>;
```

Comment:
nit: perhaps we could follow the "theme" naming and have something like `themeOverrides`.

Reply:
I agree with Carlos, I think `themeOverrides` is more consistent